extensions/terminal-suggest/src/test/fig.test.ts

```diff
@@ -23,7 +23,12 @@ export const figGenericTestSuites: ISuiteSpec[] = [
 			{ input: 'foo|', expectedCompletions, expectedResourceRequests: { type: 'both', cwd: testPaths.cwd } },
 
 			// Basic arguments (fallback)
-			{ input: 'foo |', expectedCompletions: [], expectedResourceRequests: { type: 'both', cwd: testPaths.cwd } }
+			{ input: 'foo |', expectedCompletions: [], expectedResourceRequests: { type: 'both', cwd: testPaths.cwd } },
+
+			// New lines to simulate wrapping
+			{ input: 'foo \n|', expectedCompletions: [], expectedResourceRequests: { type: 'both', cwd: testPaths.cwd } },
+			{ input: 'f\noo|', expectedCompletions, expectedResourceRequests: { type: 'both', cwd: testPaths.cwd } },
```

Comment:
Isn't input what the prompt input model hands off, therefore `f\noo` would mean it's a continuation and a real `\n`?